docs/before/analysis.md

```diff
@@ -0,0 +1,155 @@
+```bash
+export PATH=/cds/sw/package/autosfx/btx/scripts:$PATH
+```
+or for analysis on S3DF:
+```bash
+export PATH=/sdf/group/lcls/ds/tools/btx/scripts:$PATH
+```
+This change can be applied by either running `source ~/.bashrc` or opening a new terminal.
+
+We recommend always launching analysis tasks from the same directory within your scratch directory so that all the SLURM log files accumulate at that specific path. For example, we typically use the following hierarchy:
+```bash
+${SIT_PSDM_DATA}/{instrument}/{exp}/scratch/btx/
+├── yaml/
+|  └── default_config.yaml  # Experiment YAML file
+└── launchpad # <- Launch files from here
+```
+If launching from within the `launchpad` directory, as above, a specific task can be run as follows:
+```bash
+elog_submit.sh -c ../yaml/default_config.yaml -t <task> -n <ncores>
+```
+If running this from the psana cluster, the following arguments should be added to the above command: `-f SLAC -q psanaq`. Command-line arguments for the `elog_submit.sh` script follow:
+```
+| Flag  | Meaning                                             | Usage                |
+| -f    | Facility: SRCF_FFB (FFB), SLAC (psana), or S3DF     | -f <facility>        |
+| -q    | Queue to use.                                       | -q <queue>           |
+| -n    | Number of cores (soon to be deprecated)             | -n <ncores>          |
+| -a    | SLURM account, only relevant for S3DF               | -a <account>         |
+| -t    | Task to run. Must match the name in the YAML file.  | -t <task>            |
+| -c    | Path to the configuration file in YAML format.      | -c <config_file>     |
+| -e    | Name of the experiment.                             | -e <experiment_name> |
+| -r    | Run to process. If supplied override YAML value.    | -r <run_number>      |
+```
+A related executable, `submit_range_run.sh`, will launch the same task across a series of runs. This script can be run in the same fashion as `elog_submit.sh` and with the same flags, except an additional `-s` flag should be added that determines the last run number to process. 
+
+More details about available tasks can be found in the relevant experiment pages.
+
+
+#### Chaining tasks: launching workflows from the eLog
+
+The figure below illustrates an example of how tasks can be chained as workflows, which can be triggered from the eLog to be executed at S3DF with results displayed back in the eLog.
+In your browser, you can access your {experiment} logbook (eLog) at the following URL: [https://pswww.slac.stanford.edu/lgbk/lgbk/{experiment}](). 
+
+| ![btx worfklow](images/btx_sfx_workflow.png) | 
+|:--:| 
+| *__Example of SFX workflow implemented with `btx`.__ 1) Define the Airflow workflow within `btx`, eventually making use of `tppwrw` to deploy containers. 2) Setup the workflows in the eLog. 3) Trigger the workflows manually or automatically from the eLog. 4) Check the results interactively in the eLog Summary panel.* |
```

Comment:
Great figure, thanks!

I think that what is meant by step 1 could use more explanation in the actual text.